content/en/building/targets/targets-js.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "targets.js"
-linkTitle: "targets.js"
-weight: 5
+title: "Targets.js"
```

Comment:
I suggest we keep it as `target.js`, as that is the name of the file to be configured.